### ISSUE TITLE
RFC: [Core Team] Standardise English dialect

### ISSUE THREAD

**Issue by RLittlesII** [reactiveui/rfcs#11]
# Standardize English dialect usage for ReactiveUI

## Summary
While working on the [ReactiveUI Website](https://github.com/reactiveui/website), an issue was pointed out about the spelling of a word https://github.com/reactiveui/website/issues/90.  This started a conversation that I am sure has happened in the developer community for years.  What are is the correct spellings of words across the [English Dialects](https://en.wikipedia.org/wiki/English_language#Dialects,_accents,_and_varieties).

## Motivation
When dealing with a globally distributed project, and english is not everyones first language, this becomes a challenge.  For native English speakers it is challenge enough to spell certain words correctly.  Add dialect into the mix and English becomes more confusing, as english is a vary colourful language.  I believe that standardizing our publicly facing content will go a long way in helping consumption of documentation and communications.

## Discussion
I would like the @reactiveui/core-team and @reactiveui/learning-team to come together and select an english dialect for spellings moving forward to reduce friction and consumption for non native english speakers.

#### Side Note
Notice in this communication I explicitly used the same word spelled different ways to point out the confusion.

**Comment by glennawatson** [comment]
Most large projects this scale we usually just set on American English. For tech projects it seems to be the standard.

**Comment by olevett** [comment]
As a British English speaker, I'm pro the use of 'u's in words, but think that American English is probably the most sensible default...

**Comment by ghuntley** [comment]
We have a content style-guide over at https://reactiveui.net/contribute/content-style-guide/ 👍 It's based off the work that mailchimp open-sourced All content should be shaped based off the contents of the styleguide. The styleguide is the source of truth for `color` vs `colour` and how content is authored. If you think a section should be changed or burned in the style guide then send in the PR. 

Okay so my personal rule of thumb I typically use is `Colour` in documentation and `Color` in the API.

**Comment by glennawatson** [comment]
Yeah we should probably override that style guide with whatever we decide from this RFC. At the moment it's just "simple english"
